.github/workflows/tests_lgpu_python.yml

```diff
@@ -254,7 +254,7 @@ jobs:
       - name: Download coverage reports
         uses: actions/download-artifact@v4
         with:
-          pattern: ubuntu-codecov-*
+          pattern: ubuntu-codecov*
```

Comment:
Unless the file generated here does not have the `-` after codecov it is safer to keep it, because sometimes there is a directory called `ubuntu-codecov` that breaks the process.

Reply:
Not that I disagree, but this is consistent with other codecov patterns in the rest of recipes. Let me fix this in all places then